advisories/unreviewed/2022/04/GHSA-4h33-v95w-8vh4/GHSA-4h33-v95w-8vh4.json

```diff
@@ -1,20 +1,36 @@
 {
   "schema_version": "1.3.0",
   "id": "GHSA-4h33-v95w-8vh4",
-  "modified": "2022-04-09T00:00:42Z",
+  "modified": "2022-10-19T04:18:16Z",
   "published": "2022-04-03T00:01:04Z",
   "aliases": [
     "CVE-2020-25691"
   ],
-  "details": "A flaw was found in darkhttpd. Invalid error handling allows remote attackers to cause denial-of-service by accessing a file with a large modification date. The highest threat from this vulnerability is to system availability.",
+  "summary": "darkhttpd: large mtime crash",
+  "details": "A flaw was found in darkhttpd. Invalid error handling allows remote attackers to cause denial-of-service by accessing a file with a large modification date (year 10,000+). The highest threat from this vulnerability is to system availability.",
   "severity": [
     {
       "type": "CVSS_V3",
-      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
+      "score": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
     }
   ],
   "affected": [
-
+    {
+      "package": {
+        "ecosystem": "Packagist",
```

Comment:
This is wrong but I can't submit the form without this. :(

Reply:
There is a reason for that. We only support advisory mappings to a [supported set of ecosystems](https://github.com/github/advisory-database/#supported-ecosystems). Looking at your project it seems like it's expected for the end user to build the project themselves which wouldn't fit our data model. 

Thanks for the thought though :)

Reply:
Yeah I guess there isn't really an ecosystem for C/C++. :)

Okay, I'll close out the PR.